handsontable/angular-handsontable
# Feature/packaging angular

### Context
This is an idea, how to support PRO and CE versions in one wrapper.
Wrapper uses https://github.com/dherges/ng-packagr and is based on https://github.com/dherges/ng-packaged.

### How has this been tested?
1. Download this branch (`feature/packaging-angular`)
2. Run `npm run build:lib`
3. Release package in your local npm registry
4. Create a new app (https://angular.io/guide/quickstart)
5. Install the package from local npm registry
6. Install necessary Handsontable (CE/PRO)
7. Import `HotTableModule` to `app.module.ts`
8. Add `<hot-table></hot-table>` to an template.
9. `serve`, `build`, `build --prod` and `build --prod --aot` should works

### Types of changes


### Related issue(s):
1. #48

### Checklist:
